library/core/src/main/java/com/google/android/exoplayer2/text/ssa/SsaStyle.java

```diff
@@ -83,12 +85,16 @@
   public static final int SSA_ALIGNMENT_TOP_CENTER = 8;
   public static final int SSA_ALIGNMENT_TOP_RIGHT = 9;
 
+  public static final int SSA_COLOR_UNKNOWN = -1;
```

Comment:
Yeah, I missed it.. big time :) I'll fix it similarly how the `windowColorSet` is solved.

Reply:
In the end I've decided to go with a dedicated `SsaColor` class which holds this state, because we'll have multiple color attributes in the style which would lead to having a lot of `xyColorSet` flags. What do you think?